dias/downloader.py

```diff
@@ -0,0 +1,132 @@
+# Parallel Sentinel-2 downloader for DIAS
```

Comment:
Perhaps it would also be a good idea to only have the main_engine in the dias directory, and everything else in subdirectories?

This should make the entrypoint more clear.